packages/compose/README.md

```diff
@@ -315,11 +315,11 @@ with the same node.
 
 _Parameters_
 
--   _refs_ `Array<RefObject|RefCallback>`: The refs to be merged.
+-   _refs_ `Array<T>`: The refs to be merged.
```

Comment:
What is `T`?

Reply:
T is the type of the ref. I could rename it to `TRef` if that would help? I wish we were able to pull out the type parameters into the generated docs, maybe something to add to the docgen feature request list.

Reply:
I agree that having a more explicit name would be better — `TRef` or `RefType` maybe?

Reply:
`T` is very common in the TS world, but yeah, I think having a bit more specific names can improve readability.

Reply:
What does `T` stand for? Type? Could we avoid abbreviations? What about `RefType`? Isn't that much clearer?

Reply:
T stands for Type yes, its the canonical abbreviation for a generic type. Adding T to the front generally indicates its a generic type as well, so TRef is appropriate is more appropriate than `RefType` which could easily be confused for an actual type rather than a generic type parameter.